packages/form-core/test-helpers/getFormControlMembers.js

```diff
@@ -12,7 +12,7 @@ export function getFormControlMembers(el) {
   // eslint-disable-next-line
   const { _inputNode, _helpTextNode, _labelNode, _feedbackNode, _allValidators } = el;
   return {
-    _inputNode,
+    _inputNode: /** @type {* & HTMLInputElement} */ (el)._inputNode,
```

Comment:
I think it should be `/** @type {* & HTMLInputElement} */ (el._inputNode)`?

Reply:
Nope, the type was for the `el` element. But i changed it into `FormControlHost`